src/main/java/com/microsoft/graph/requests/extensions/DriveItemRequestBuilder.java

```diff
@@ -195,8 +200,8 @@ public IDriveItemCreateLinkRequestBuilder createLink(final String type, final St
         return new DriveItemCreateLinkRequestBuilder(getRequestUrlWithAdditionalSegment("microsoft.graph.createLink"), getClient(), null, type, scope, expirationDateTime, password, message);
     }
 
-    public IDriveItemCreateUploadSessionRequestBuilder createUploadSession(final DriveItemUploadableProperties item) {
-        return new DriveItemCreateUploadSessionRequestBuilder(getRequestUrlWithAdditionalSegment("microsoft.graph.createUploadSession"), getClient(), null, item);
+    public IDriveItemCreateUploadSessionRequestBuilder createUploadSession(final DriveItemUploadableProperties item, final Boolean deferCommit) {
```

Comment:
`deferCommit` needs to be optional - https://docs.microsoft.com/en-us/graph/api/driveitem-createuploadsession?view=graph-rest-beta#create-an-upload-session. That's why the `OneDriveTests` is failing.